README-PyPi.md

```diff
@@ -128,4 +128,4 @@ There are a few settings we recommend to edit in your user settings.
 - Privacy policy: [https://quantum.ibm.com/terms/privacy](https://quantum.ibm.com/terms/privacy)
 - Cloud Services Agreement [https://www.ibm.com/support/customer/csol/terms/?id=Z126-6304&cc=us&lc=en](https://www.ibm.com/support/customer/csol/terms/?id=Z126-6304&cc=us&lc=en)
 - IBM Cloud Service Description [https://www.ibm.com/support/customer/csol/terms/?id=i126-6605&lc=en](https://www.ibm.com/support/customer/csol/terms/?id=i126-6605&lc=en)
-- EULA acceptance required before starting to use the model
+- [End User License Agreement (EULA)](docs/EULA.md) acceptance required before starting to use the model acceptance required before starting to use the model
```

Comment:
since this doc is for publishing on PyPi this link needs to be static not relative, most likely this (or close to it): `https://github.com/Qiskit/qiskit-code-assistant-jupyterlab/blob/main/docs/EULA.md`

Reply:
Right, done https://github.com/Qiskit/qiskit-code-assistant-jupyterlab/pull/4/commits/96660c55415331bc2825c32ce0e133fe9d15ad66